### DISCUSSION TITLE
Add provided.al2023 for Lambda Runtime

### DISCUSSION THREAD

**Discussion by markrmaci** [aws-cloudformation/cfn-lint#3046] [Ideas]
Add the value 'provided.al2023' to the list of lambda runtimes.

**Comment by kddejong** [comment]
Its there.  What version of cfn-lint are you using?  You probably just need to do an update.

**Comment by markrmaci** [comment]
Sorry my mistake. Apparently the vscode extension must use its own values/schema for this. Please ignore. I do have the latest cfn-lint package.
